FogCreek/Glitch-Community
# Standing statistic: bumping version number of shared-components

## Links
* Remix link: https://standing-statistic.glitch.me/
* Issue-tracker link: https://app.clubhouse.io/glitch/story/3548/clicking-on-an-interactive-element-does-not-get-focus-if-a-popover-is-currently-open [ch3548]

## GIF/Screenshots:
see screenshot/explanation here: https://github.com/FogCreek/shared-components/pull/61

## Changes:
* bumps to latest shared-components. In theory this shouldn't be necessary since we have the `^` in front of the version, but I couldn't get it to work without doing this 🤷‍♀ 

## How To Test:
* click on a popover, then click on another popover while the first is still open, you're focus should be on the new popover
